base_layer/transaction_components/src/key_manager/interface.rs

```diff
@@ -187,8 +188,7 @@ pub trait TransactionKeyManagerInterface: Clone + Send + Sync + 'static {
         sender_offset_key_ids: &[TariKeyId],
     ) -> Result<PrivateKey, KeyManagerError>;
 
-    // Look into perhaps removing all nonce here, if the signer and receiver are the same it should not be required to
-    // share or pre calc the nonces
+    // creates a metadata signature for the output without requiring manual user verification on a ledger device
```

Comment:
Please capitalise all comments - we actually need a clippy rule enforcing sentence case for comments.